### PR TITLE
Add support of /folder/folder_handler#folder_key/file/file_handle urls

### PR DESCRIPTION
**Pull request by edtechd**
Support of the following urls should be added
https://mega.nz/folder/folder_handler#folder_key/file/file_handler

However downloading of such items is not possible. What do you think, is there any way to make it working?

### CODE REVIEW

**Comment by qgustavor** [comment]
Looking at the code seems there's nothing wrong. I'll test it to find what's happening.

File: lib/file.js [hunk 1]
```diff
@@ -358,26 +358,28 @@ class File extends EventEmitter {
...
 
     if (url.path.match(/\/(file|folder)\//) !== null) {
       // new format
+      const split = url.hash.substr(1).split('/file/')
       const fileHandker = url.path.substring(
         url.path.lastIndexOf('/') + 1,
         url.path.length + 1
       )
-      const fileKey = url.hash.substr(1)
+      const fileKey = split[0].substr(1)
```
Comment (qgustavor) [inline lib/file.js:374]: `.substr(1)` is being called twice.

```suggestion
      const fileKey = split[0]
```

**Comment by edtechd** [comment]
Done. I just double checked it, and it seems to be working.

**Comment by qgustavor** [comment]
I will publish it in NPM.

**Pull request merged**